src/Peachpie.Library.Network/CURLFunctions.cs

```diff
@@ -233,7 +233,7 @@ internal static IEnumerable<string> CookiesToNetscapeStyle(CookieCollection cook
 
         static PhpArray CreateCookiePhpArray(CookieCollection cookies)
         {
-            return new PhpArray(CookiesToNetscapeStyle(cookies));
+            return (cookies != null ? new PhpArray(CookiesToNetscapeStyle(cookies)) : PhpArray.Empty);
```

Comment:
maybe check `null` in `CookiesToNetscapeStyle` instead?

Reply:
@jakubmisek I'd appreciate it if you could edit this pull request to reflect the changes for this one